vision/google/cloud/vision/annotations.py

```diff
@@ -0,0 +1,130 @@
+    FeatureTypes.SAFE_SEARCH_DETECTION: 'safeSearchAnnotation',
+    FeatureTypes.TEXT_DETECTION: 'textAnnotations',
+}
+
+
+class Annotations(object):
+    """Annotation class for managing responses.
+
+    :type faces: list
+    :param faces: List of :class:`~google.cloud.vision.face.Face`.
+
+    :type properties: list
+    :param properties:
+        List of :class:`~google.cloud.vision.color.ImagePropertiesAnnotation`.
+
+    :type labels: list
+    :param labels: List of
+                   :class:`~google.cloud.vision.entity.EntityAnnotation`.
+
+    :type landmarks: list
+    :param landmarks: List of
+                      :class:`~google.cloud.vision.entity.EntityAnnotation.`
+
+    :type logos: list
+    :param logos: List of
+                  :class:`~google.cloud.vision.entity.EntityAnnotation`.
+
+    :type safe_searches: list
+    :param safe_searches:
+        List of :class:`~google.cloud.vision.safe.SafeSearchAnnotation`
+
+    :type texts: list
+    :param texts: List of
+                  :class:`~google.cloud.vision.entity.EntityAnnotation`.
+    """
+    def __init__(self, faces=None, properties=None, labels=None,
+                 landmarks=None, logos=None, safe_searches=None, texts=None):
+        self.faces = faces or []
+        self.properties = properties or []
+        self.labels = labels or []
+        self.landmarks = landmarks or []
+        self.logos = logos or []
+        self.safe_searches = safe_searches or []
+        self.texts = texts or []
+
+    @classmethod
+    def from_api_repr(cls, response):
+        """Factory: construct an instance of ``Annotations`` from a response.
+
+        :type response: dict
+        :param response: Vision API response object.
+
+        :rtype: :class:`~google.cloud.vision.annotations.Annotations`
+        :returns: An instance of ``Annotations`` with detection types loaded.
+        """
+        annotations = {}
+
+        for feature_type in response.keys():
+            annotations[feature_type] = []
+
+        for feature_type, annotation in response.items():
+            annotations[feature_type].extend(
+                _entity_from_response_type(feature_type, annotation))
+
+        faces = annotations.get(
+            _REVERSE_TYPES[FeatureTypes.FACE_DETECTION], [])
+        properties = annotations.get(
+            _REVERSE_TYPES[FeatureTypes.IMAGE_PROPERTIES], [])
+        labels = annotations.get(
+            _REVERSE_TYPES[FeatureTypes.LABEL_DETECTION], [])
+        landmarks = annotations.get(
+            _REVERSE_TYPES[FeatureTypes.LANDMARK_DETECTION], [])
+        logos = annotations.get(
+            _REVERSE_TYPES[FeatureTypes.LOGO_DETECTION], [])
+        safe_searches = annotations.get(
+            _REVERSE_TYPES[FeatureTypes.SAFE_SEARCH_DETECTION], [])
+        texts = annotations.get(
+            _REVERSE_TYPES[FeatureTypes.TEXT_DETECTION], [])
+
+        return cls(faces=faces, properties=properties, labels=labels,
```

Comment:
If you used approprate key names in your annotations dict, then you could just do

```
return cls(**annotations)
```